test/mock/grpc/cmd/main.go

```diff
@@ -17,9 +17,31 @@ import (
 )
 
 func main() {
+	var configDirectory string
 	var grpcAddress string
 	var apiAddress string
 
+	currentPath, err := os.Getwd()
+	if err != nil {
+		slog.Error("Unable to get current directory", "error", err)
```

Comment:
This may be overkill for a mock server but should we specify a fallback for this in case we get an error (which should never really happen)? e.g. something like `/tmp/mock-management-plane`?

Reply:
Getting the current directory is a fallback already. Its the default used in case where the user doesn't specify the `configDirectory` command line argument. I think it might be overkill alright adding a fallback. If it errors saying cant get current directory then its up to the user to specify a directory via the command line argument.

Reply:
Fair, let's leave it as is 👍